src/project.coffee

```diff
@@ -68,9 +68,10 @@ class Project extends Model
   deserializeParams: (params) ->
     params.buffers = _.compact params.buffers.map (bufferState) ->
       # Check that buffer's file path is accessible
+      return if fs.isDirectorySync(bufferState.filePath)
```

Comment:
This makes me nervous in terms of speed. Now for all open files, we need to hit the disk twice: check if dir, and check if we can read it.

I needed to do this as opening with only `r` does not throw an error when there is a directory.